src/components/TodoList/TodoList.tsx

```diff
@@ -40,16 +40,17 @@ export const TodoList: React.FC<Props> = ({
             })}
           >
             <td className="is-vcentered">{todo.id}</td>
-            {!todo.completed && (
-              <td className="is-vcentered" />
-            )}
-            {todo.completed && (
+
+            {todo.completed ? (
               <td className="is-vcentered">
                 <span className="icon" data-cy="iconCompleted">
                   <i className="fas fa-check" />
                 </span>
               </td>
+            ) : (
+              <td className="is-vcentered" />
             )}
```

Comment:
i'd render the span conditionally :)
```suggestion
              <td className="is-vcentered">
                {todo.completed && (
                  <span className="icon" data-cy="iconCompleted">
                    <i className="fas fa-check" />
                  </span>
                )}
              </td>
```